firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -82,45 +82,36 @@ void home()
 
 // Moves the magnet from the "start" point to the "end" point
 // This can only move in straight lines
-// Returns a boolean indicating success/error
-bool moveStraight(int motor[], float startCol, float startRow, float endCol, float endRow)
+// A specific motor is passed to this function since we are only moving one here
+// Returns a code indicating error or success circumstances
```

Comment:
If you update to an enum, you can have a more descriptive message about the return value like:
```c
// Returns a `MovementStatus` code, `0` if successful, nonzero
// for all errors (see `MovementStatus` in chessboard.ino).
```